tests/PhpPact/Consumer/Service/MockServerTest.php

```diff
@@ -141,17 +143,8 @@ public function testCleanUp(): void
         $this->mockServer->cleanUp();
     }
 
-    protected function assertClientCalls(array $calls): void
+    protected function getTransport(bool $secure): string
     {
-        $this->client
-            ->expects($this->exactly(count($calls)))
-            ->method('call')
-            ->willReturnCallback(function (...$args) use (&$calls) {
-                $call = array_shift($calls);
-                $return = array_pop($call);
-                $this->assertSame($call, $args);
-
-                return $return;
-            });
```

Comment:
so is this assertion now in the trait, simplifying responsibility of getTransport?

Reply:
There are 2 changes that are not related to each other:
* Remove `assertClientCalls()`
* Add `getTransport()`

Because these 2 changes happen at the same time, at the same place, then git think that I replace `assertClientCalls()` by `getTransport()`, but I didn't. I removed `assertClientCalls()` first, then I add `getTransport()`, which is not related to the other one.

Probably I should add `getTransport()` to other position to make it clearer

Reply:
I think I understand your point: You want to split `assertClientCalls()` removing change into another PR?

Reply:
Will add this change back in another PR